### PR TITLE
Log the VRP summary at the end of a server validation run.

### LINKED ISSUE #616: Log the summary at the end of a run
Basically the output of `-f summary`.

### PR DESCRIPTION
**Pull request by partim**
Fixes #616.

### CODE REVIEW

File: src/output.rs [hunk 1]
```diff
@@ -816,6 +817,45 @@ impl<W: io::Write> Formatter<W> for Summary {
...
+            metrics.vrps.valid, metrics.vrps.marked_unsafe,
+            metrics.vrps.contributed,
+        ))
+    }
+
+    pub fn log(metrics: &Metrics) {
+        Self::produce_header(metrics, |args| {
+            info!("{}", args);
+            Ok(())
+        }).unwrap()
```
Comment (DRiKE) [inline src/output.rs:846]: If for some very exotic reason we can not write to stdout (or, to wherever stdout is redirected), this `unwrap` can cause trouble I guess?

File: src/output.rs [hunk 2]
```diff
@@ -792,11 +792,12 @@ impl<W: io::Write> Formatter<W> for Rpsl {
...
 
-struct Summary;
+/// Output only a summary.
+pub struct Summary;
 
-impl<W: io::Write> Formatter<W> for Summary {
-    fn header(
-        &self, _snapshot: &PayloadSnapshot, metrics: &Metrics, target: &mut W
+impl Summary {
+    pub fn write(
```
Comment (DRiKE) [inline src/output.rs:799]: is `write` still used somewhere? Looks like `log` + `produce_header` replaced it entirely.

**Review by DRiKE — APPROVED** [review]
No blocking problems, but it seems there is some leftover code (`pub fn write`) or a case of code repetition. Or I am missing something.

File: src/output.rs (hunk 1 above)
Comment (partim) [reply src/output.rs:846]: The only way that `produce_header` can return an error is if the closure returns an error, which, in this case it never does, so this is fine.

File: src/output.rs (hunk 2 above)
Comment (partim) [reply src/output.rs]: Uhm, indeed. Forgot to delete it. Fixed.

**Pull request merged**